datacapture/src/main/java/com/google/android/fhir/datacapture/QuestionnaireViewModel.kt

```diff
@@ -602,14 +592,45 @@ internal class QuestionnaireViewModel(application: Application, state: SavedStat
           ),
       )
 
-  private suspend fun updateDependentQuestionnaireResponseItems(
+  private suspend fun initializeCalculatedExpression(
```

Comment:
add some documentation - especially explain that in this funtion we go over all calculated expressions in the questionnaire and evaluate them.

```suggestion
  private suspend fun initializeCalculatedExpressions(
```